pkg/stub/imagestreams.go

```diff
@@ -503,38 +520,34 @@ func (h *Handler) processImportStatus(is *imagev1.ImageStream, cfg *v1.Config) (
 		logrus.Debugf("no error/progress checks cause stream %s is skipped", is.Name)
```

Comment:
Any reason to keep the large, commented code block?

Reply:
yeah I'd like for the fact that we have commented it out soak a bit longer @adambkaplan  ... if we have to add some form of it back, having the prior form of it here will expedite things